library/src/window_impl_noRender.cxx

```diff
@@ -16,6 +17,7 @@ window_impl_noRender::window_impl_noRender(const options& options)
   : window_impl(options)
   , Internals(new window_impl_noRender::internals)
 {
+  this->Internals->RenWin->SetOffScreenRendering(true);
```

Comment:
I don't like the fact we have an OpenGL render window instance if we have a noRender f3d window. Maybe we should implement a `vtkNoRenderWindow` inheriting from `vtkRenderWindow` and doing nothing.

Reply:
Out of scope of this PR, but that would be better indeed if possible.